### ISSUE TITLE
Cata Classic: Key Ring Still Present

### ISSUE THREAD

**Issue by Clamsoda** [xod-wow/LiteBag#54]
Cata Classic release still shows a keyring bag icon.  Clicking it expands the bag to show equipped gear it appears.  Hovering over an item produces error:

```
3x ...zard_UIPanels_Game/Classic/ContainerFrame_Shared.lua:1403: attempt to call global 'KeyRingButtonIDToInvSlotID' (a nil value)
[string "@Blizzard_UIPanels_Game/Classic/ContainerFrame_Shared.lua"]:1403: in function <...zard_UIPanels_Game/Classic/ContainerFrame_Shared.lua:1397>
[string "=[C]"]: in function `ContainerFrameItemButton_OnEnter'
[string "@LiteBag/ItemButton.lua"]:26: in function <LiteBag/ItemButton.lua:21>
```

![Keyring](https://github.com/xod-wow/LiteBag/assets/3506429/15e87663-14bb-4bb4-8787-1d4a7ae57753)

**Comment by mbattersby** [comment]
Keyring is deleted in cata classic?

**Comment by Clamsoda** [comment]
I believe so.  It definitely doesn't function as expected.

**Comment by mbattersby** [comment]
Should be fixed in 4.4.0-3-cata
